packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -246,9 +246,9 @@ const conf = (module.exports = convict({
       format: Boolean,
       env: 'REACT_CONVERSION_PAIR_ROUTES',
     },
-    postVerifyAddRecoveryKeyRoutes: {
+    postVerifyOtherRoutes: {
       default: false,
-      doc: 'Enable users to visit the React version of "post-verify add recovery key" routes',
+      doc: 'Enable users to visit the React version of any other "post verify" routes',
       format: Boolean,
       env: 'REACT_CONVERSION_POST_VERIFY_ADD_RECOVERY_KEY_ROUTES',
```

Comment:
Should the env var be renamed  to `REACT_CONVERSION_OTHER_POST_VERIFY_ROUTES` for clarity, or is that unnecessarily complicated?

Also, should this flag also be renamed in:
- packages/fxa-content-server/app/tests/spec/lib/router.js#L39
- packages/fxa-content-server/server/lib/routes/react-app/types.ts#L55
- packages/fxa-content-server/tests/server/routes/react-app.js#L41

Reply:
We've got `postVerifyCADViaQRRoutes` plus `postVerifyThirdPartyAuthRoutes` so I think it's consistent to go with `postVerifyOtherRoutes`, but you did catch where I forgot to change `REACT_CONVERSION_POST_VERIFY_ADD_RECOVERY_KEY_ROUTES` to `REACT_CONVERSION_POST_VERIFY_OTHER_ROUTES`. 😅 So thanks!